webapp/src/types/common/index.d.ts

```diff
@@ -138,3 +138,9 @@ type SubscriptionFilters = {
     serviceType: string,
     eventType: string,
 }
+
+type LabelIconProps = {
+    className: string
+    tooltipText: string
+    extraClassName?: string
```

Comment:
We can add a comment about how we're using `className` and how we are using `extraClassName`.